.travis.yml

```diff
@@ -17,6 +17,6 @@ jobs:
       env:
         - test: Editor Config check
```

Comment:
@PeterDaveHello what is this env var used for?

Reply:
Just help identify different test on the page

Reply:
Ah, cool.